srcpkgs/ruby-travis/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'ruby-travis'
+pkgname=ruby-travis
+version=1.8.9
+revision=1
+noarch=yes
+build_style=gem
+depends="ruby-backports ruby-faraday>=0.9 ruby-faraday_middleware>=0.9
+ ruby-faraday_middleware>=0.9.1 ruby-gh>=0.13 ruby-highline>=1.6
```

Comment:
Seems to me as if ruby-faraday_middleware is specified twice

Reply:
It is because the people at it declared it with

ruby-faraday_middleware >= 0.9.1, ruby-faraday_middleware >= 0.9

Reply:
Uhhh...remove the latter then, I suppose?